hermeznetwork/wallet-ui
# Replace exchange rates api

### What does this PR does?

This PR replaces the exchange rates API.

### How to test?

1. Run the app.
2. Check that all conversions to fiat are working fine.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
